player.py

```diff
@@ -0,0 +1,32 @@
+def percent_diff(a, b):
+    return ((a - b) / b) * 100
+
+
+def alert_open(results, last_trade):
+    a = results[len(results) - 1]['price']
+    b = results[len(results) - 2]['price']
+    c = results[len(results) - 3]['price']
+    if len(last_trade) == 0 or last_trade[0]['buy'] is False:
+        if percent_diff(b, c) < 0:
+            if percent_diff(a, b) >= 0.1:
```

Comment:
Recommendation: These "magic numbers" (0.1 in this case) can be declared at the start of the file. In the future they can be defined in a config or so.